util/pace/util/grid/stretch_transformation.py

```diff
@@ -90,14 +102,14 @@ def direct_transform(
         lon_transform.data[:] = lon_trans
         lat_transform.data[:] = lat_trans
 
-    elif isinstance(lon, np.ndarray):
+    elif isinstance(lon, numpy_module.ndarray):
         lon_transform = lon_trans
         lat_transform = lat_trans
 
     return lon_transform, lat_transform
 
 
-def _sign(input, pn):
+def _sign(input, pn, numpy_module):
```

Comment:
I'm kind of confused by the existence of this function. Isn't there a `np.sign` and `cp.sign`?

Reply:
Wait does this actually do `np.abs(input) * np.sign(pn)`? Can you use that instead? Why is it called `pn`?

Reply:
Wasn't aware of np.sign, that's now done.